pages/[categoryId]/index.tsx

```diff
@@ -18,6 +19,9 @@ export default function CategoryHomepage({
       </Head>
       <h1 className="page-title">All {categoryInfo.name}</h1>
       <CategoryNav categoryInfo={categoryInfo} />
+      <Notice title="Can't find what you're looking for?">
```

Comment:
Might this look nicer at the bottom of the page?

Reply:
Good suggestion, thanks! Have moved.